AUTHORS

```diff
@@ -13,23 +15,27 @@ Andrey Voronkov                         voronkovaa@gmail.com
 Anit Gandhi                             anitgandhi@gmail.com
 Archana Shinde                          archana.m.shinde@intel.com
 Arika Chen                              eaglesora@gmail.com
+Arthur Chiao                             arthurchiao@hotmail.com
```

Comment:
There's an extra space in front of the email. Not sure if that's bad or not, or even how it got there :shrug:

Reply:
I added a space in the "ArthurChiao" name :)